tarantool/connection.py

```diff
@@ -763,33 +765,33 @@ def upsert(self, space_name, tuple_value, op_list, **kwargs):
         '''
         Execute UPSERT request.
 
-        If there is an existing tuple which matches the key fields of
+        If an existing tuple matches the key fields of
         `tuple_value`, then the request has the same effect as UPDATE
         and the [(field_1, symbol_1, arg_1), ...] parameter is used.
 
-        If there is no existing tuple which matches the key fields of
+        If there is no tuple matching the key fields of
         `tuple_value`, then the request has the same effect as INSERT
         and the `tuple_value` parameter is used. However, unlike insert
-        or update, upsert will not read a tuple and perform error checks
+        or update, upsert will neither read the tuple nor perform error checks
         before returning -- this is a design feature which enhances
         throughput but requires more caution on the part of the user.
 
-        If you're using secondary index, it must be unique.
+        If you're using a secondary index, it must be unique.
 
-        List of operations allows to update individual fields.
+        The list of operations allows updating individual fields.
+        
+        For every operation, you must provide the field number to apply this
+        operation to.
 
         *Allowed operations:*
 
-        (For every operation you must provide field number, to apply this
-        operation to)
-
         * `+` for addition (values must be numeric)
         * `-` for subtraction (values must be numeric)
         * `&` for bitwise AND (values must be unsigned numeric)
         * `|` for bitwise OR (values must be unsigned numeric)
         * `^` for bitwise XOR (values must be unsigned numeric)
-        * `:` for string splice (you must provide `offset`, `count` and `value`
-          for this operation)
+        * `:` for string splice (you must provide `offset`, `count`,
+           and `value` for this operation)
```

Comment:
```suggestion
          and `value` for this operation)
```
The markup is broken due to the extra space.
![list](https://user-images.githubusercontent.com/29621138/177329773-7fa5e854-3e82-40bd-aaeb-872888de4af1.png)

Reply:
Thank you!